react-relay/index.d.ts

```diff
@@ -128,7 +146,13 @@ declare module "react-relay" {
     }
 
     interface RelayProp {
-        variables: any
-        setVariables(variables: Object, onReadyStateChange?: OnReadyStateChange): void
+        route: { name: string; }; // incomplete, also has params and queries
+        variables: any;
+        pendingVariables?: any;
+        setVariables(variables: Object, onReadyStateChange?: OnReadyStateChange): void;
+        forceFetch(variables: Object, onReadyStateChange?: OnReadyStateChange): void;
```

Comment:
I was copy-pasting from elsewhere in the type file. Happy to make all of them into `Object`; as that is more accurate.